main.rb

```diff
@@ -51,6 +58,23 @@ class Saying
   { saying: Saying.all.sample }.to_json
 end
 
+get '/slackit/?' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.